### PR TITLE
Add json parse file in utils

### PR DESCRIPTION
**Pull request by chilianyi**

### CODE REVIEW

**Comment by chilianyi** [comment]
Add dump json to file func. Yes, it is just a func wrap, may be there are some changes later, such as need to replace some chars when load.

File: pkg/utils/json.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+package utils
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	"openpitrix.io/openpitrix/pkg/logger"
+)
+
+func Loads(src string, dest interface{}) error {
```
Comment (chai2010) [inline pkg/utils/json.go]: loads is not a good name in this package, maybe loadsxxx is better

File: pkg/utils/json.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+func Load(filename string, dest interface{}) error {
+	data, err := ioutil.ReadFile(filename)
+	if err != nil {
+		logger.Errorf("Failed to load json from file [%s]: %v", filename, err)
+		return err
+	}
+	return Loads(string(data), &dest)
+}
+
+func Dumps(src interface{}) (string, error) {
```
Comment (chai2010) [inline pkg/utils/json.go]: ditto

File: pkg/utils/json.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	"encoding/json"
+	"io/ioutil"
+
+	"openpitrix.io/openpitrix/pkg/logger"
+)
+
+func Loads(src string, dest interface{}) error {
+	err := json.Unmarshal([]byte(src), &dest)
+	if err != nil {
+		logger.Errorf("Failed to load json: %v", err)
```
Comment (chai2010) [inline pkg/utils/json.go]: it is just a little function, the caller will do it when some error occurred

File: pkg/utils/json.go (hunk 1 above)
Comment (chilianyi) [reply pkg/utils/json.go]: It just keep the same with standard json lib we used in python. 
http://www.runoob.com/python/python-json.html

File: pkg/utils/json.go (hunk 3 above)
Comment (chilianyi) [reply pkg/utils/json.go]: The caller no need to logger again, just use this func is enough. With this log, we know an error exist, sometimes the error is not that critical.

**Pull request closed without merging**